src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimatorMeanIterative.h

```diff
@@ -370,7 +376,7 @@ namespace OpenMS
         }
 
         // store result
-        stn_estimates_[*window_pos_center] = (*window_pos_center).getIntensity() / noise;
+        stn_estimates_[window_count] = ((*window_pos_center).getIntensity() / noise);
```

Comment:
Why has this line been changed? I don't see a change in functionality.

Reply:
The type of stn_estimates_ is map, map<PeakType, double>, so I used the iteration(*window_pos_center) to assign it.
But the type of stn_estimates now is vector, so I used a index(int window_count) instead.